components/layout/Navbar.tsx

```diff
@@ -140,28 +131,27 @@ export function Navbar({ }) {
             </a>
           </Link>
           <div className="w-full flex justify-end items-center gap-4">
+            <Button color="white" size="small" onClick={openModal}>
+              <MdEmail className="mr-2"/>
+              Subscribe
+            </Button>
             <LinkButton
-              link="https://app.tina.io/signin"
-              color="white"
-              size="small"
-            >
-              Log In
-            </LinkButton>
-            <LinkButton
-              link="https://app.tina.io/register"
+              link="https://meetings-eu1.hubspot.com/scott15?uuid=2f12f1a3-8b6a-4472-bfb5-2f3f930a1b19"
               color="blue"
               size="small"
             >
-              Sign Up
+              <FaCalendarDay className="mr-2"/>
+              Book a Demo
             </LinkButton>
           </div>
         </div>
         <div
           ref={navRef}
-          className={`absolute ${stuck
-            ? `min-[1135px]:fixed shadow-sm bg-gradient-to-r from-[rgba(216,251,248,0.6)] to-[rgba(215,233,255,0.6)] backdrop-blur animate-slide-in top-0 p-4`
-            : `translate-y-2 px-4 pt-4 pb-6`
-            } z-40 w-full min-[1135px]:px-10 hidden min-[1135px]:flex items-center justify-between gap-6`}
+          className={`absolute ${
+            stuck
+              ? `min-[1135px]:fixed shadow-sm bg-gradient-to-r from-[rgba(216,251,248,0.6)] to-[rgba(215,233,255,0.6)] backdrop-blur animate-slide-in top-0 p-4`
+              : `translate-y-2 px-4 pt-4 pb-6`
+          } z-40 w-full min-[1135px]:px-10 hidden min-[1135px]:flex items-center justify-between gap-6`}
```

Comment:
lots of arbitrary values in here - definitely try get these into tailwind config

Reply:
this was here prior - do you still want me to move it into tailwind config - the main thing is is the coloring but this style is used throughout

Reply:
> this was here prior

Ok, make a tech debt PBI for this then 
https://www.ssw.com.au/rules/technical-debt/

Reply:
https://github.com/tinacms/tina.io/issues/1823